src/test/java/de/orchound/gameoflife/game/BoardTest.java

```diff
@@ -39,4 +39,22 @@ void testUpdate() {
 		assertFalse(board.target[1][2]);
 		assertFalse(board.target[3][2]);
 	}
+
```

Comment:
Move your comment here as a block comment:
```suggestion
	/**
	 * Verifies that the top row is treated as neighbors to the bottom row,
	 * and the right edge is treated as a neighbor to the left edge.
	 */
```